Cargo.toml

```diff
@@ -7,12 +7,16 @@ members = [
     "erc20-market-cap",
     "erc721",
     "substreams-helper",
-    "uniswap-v2"
+    "uniswap-v2",
 ]
+exclude = ["messari-cli"]
 
 [workspace.dependencies]
 substreams = "~0.0.20"
-substreams-ethereum = "0.2.2-rc.2"
+-substreams-ethereum = "0.2.2-rc.2"
+
+[build]
+target = "wasm32-unknown-unknown"
```

Comment:
This is just moving the build requirement from .cargo/config.toml to ./Cargo.toml. This is due to all projects within the outer repo inheriting the build requirements from .cargo/config.toml - even the projects that are excluded from the workspace like messari-cli. Once the build target is moved to ./Cargo.toml it allows projects outside the workspace like messari-cli to be able to build with default targets rather than the workspace target.